crypto/s2n_pkey.c

```diff
@@ -129,14 +129,29 @@ int s2n_pkey_free(struct s2n_pkey *key)
     return S2N_SUCCESS;
 }
 
-int s2n_asn1der_to_private_key(struct s2n_pkey *priv_key, struct s2n_blob *asn1der)
+int s2n_asn1der_to_private_key(struct s2n_pkey *priv_key, struct s2n_blob *asn1der, int type_hint)
```

Comment:
bummer that they didn't use an enum here...

Reply:
Yeah. I considered using an internal one, and that would have avoided the CBMC issues. But we'd just have to immediately convert back to the int to use the enum.